pkg/agent/types/checker.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 Antrea Authors
```

Comment:
Should we name the file "secondarynetwork.go"? "checker.go" is too generic.

Reply:
Renamed